tools/common/Target.cs

```diff
@@ -572,7 +572,7 @@ void GenerateMacMain (StringWriter sw)
 			}
 			if (!App.IsDefaultMarshalManagedExceptionMode)
 				sw.WriteLine ("\txamarin_marshal_managed_exception_mode = MarshalManagedExceptionMode{0};", App.MarshalManagedExceptions);
-			sw.WriteLine ("\txamarin_marshal_objectivec_exception_mode = MarshalObjectiveCExceptionMode{0};", App.MarshalObjectiveCExceptions);
+			sw.WriteLine ("\txamarin_marshal_objectivec_exception_mode = MarshalObjectiveCExceptionMode{0};", "Disable" /* https://github.com/dotnet/runtime/issues/43204 App.MarshalObjectiveCExceptions */);
```

Comment:
is that code limited to net6 nugets ?
or is it used by XM packages ?

Reply:
🤦